core/dbt/graph/selector.py

```diff
@@ -380,18 +388,47 @@ def get_nodes_from_spec(self, graph, spec):
 
         return collected
 
-    def select_nodes(self, graph, raw_include_specs, raw_exclude_specs):
-        selected_nodes: Set[str] = set()
+    def get_nodes_from_multiple_specs(
+            self,
+            graph,
+            specs,
+            nodes=None,
+            check_existence=False,
+            exclude=False
+    ):
+        selected_nodes: Set[str] = coalesce(nodes, set())
+        operator = set.difference_update if exclude else set.update
 
-        for raw_spec in split_specs(raw_include_specs):
+        for raw_spec in split_specs(specs):
             spec = SelectionCriteria(raw_spec)
-            included_nodes = self.get_nodes_from_spec(graph, spec)
-            selected_nodes.update(included_nodes)
+            nodes = self.get_nodes_from_spec(graph, spec)
 
-        for raw_spec in split_specs(raw_exclude_specs):
-            spec = SelectionCriteria(raw_spec)
-            excluded_nodes = self.get_nodes_from_spec(graph, spec)
-            selected_nodes.difference_update(excluded_nodes)
+            if check_existence:
+                alert_non_existence(raw_spec, nodes)
+
+            operator(selected_nodes, nodes)
+
+        return selected_nodes
+
+    def select_nodes(self, graph, raw_include_specs, raw_exclude_specs):
+        raw_exclude_specs = coalesce(raw_exclude_specs, [])
+        check_existance = True
+
+        if not raw_include_specs:
+            check_existance = False
+            raw_include_specs = ['fqn:*', 'source:*']
+
+        selected_nodes = self.get_nodes_from_multiple_specs(
+            graph,
+            raw_include_specs,
+            check_existence=check_existance
```

Comment:
Super nit, but can you change all the instances of `check_existance` to `check_existence`? I see both!

Reply:
Ohh. Sorry!